src/Api/ContentApi.php

```diff
@@ -459,6 +460,118 @@ private function fetchDatasourceEntriesResultPage (
 		}
 	}
 
+
+	/**
+	 * Fetches all links.
+	 *
+	 * @throws ContentRequestFailedException
+	 *
+	 * @return array<StoryblokLink>
+	 */
+	public function fetchAllLinks (
+		ReleaseVersion $version = ReleaseVersion::PUBLISHED,
+	) : array
+	{
+		// force per_page to the maximum to minimize pagination
+		$query["per_page"] = 1000;
+		$query["version"] = $version->value;
+		$query["cv"] = $this->getSpaceInfo()->getCacheVersion();
```

Comment:
Please do not implicitly create a new `$query` array. Since we don't have a `$query` argument being passed-in, we can fully create this array.